source/Windows/Overlay/OverlayView.xaml

```diff
@@ -165,7 +165,7 @@
                 <TextBox Grid.Row="0" Grid.Column="3" Text="iLvl" />
                 <TextBox Grid.Row="0" Grid.Column="4" Text="Age" HorizontalAlignment="Center" />
 
-                <ScrollViewer Grid.Row="1" Grid.ColumnSpan="5" VerticalScrollBarVisibility="Auto">
+                <ScrollViewer Name="_itemList" Grid.Row="1" Grid.ColumnSpan="5" VerticalScrollBarVisibility="Auto">
```

Comment:
Would you mind renaming this binding to `ItemListingControlsScrollViewer`? It might not be the correct way to name things in WPF, but at least it will follow what's already there and we can refactor it easily later.

Reply:
You know what, who cares lol.

Reply:
No problem use the naming convention you most like. 
Feel free to change it yourself since you merged already :)